lib/static_sync/processor.rb

```diff
@@ -27,7 +28,7 @@ def sync
               log.info("Uploading #{file}") if @config.log
             end
             begin
-              @storage.create(current_file.headers)
+              @storage.create(current_file.headers) unless skip(file)
```

Comment:
Yet another branch in this (already complex) method makes it hard to follow the possible outcomes. How can we simplify this?